### PR TITLE
Reorganize docs, examples and repository

### PR DESCRIPTION
**Pull request by airadier**
* Added Dockerfile_v1 file to build images for v1 (backward compatibility)
* Add a Jenkins example for V1
* Restore the contents of v1.md from the old V1 README
* Create version_v1 file and Makefile_v1 to manage v1 version releases

### CODE REVIEW

**Review by nestorsalceda — APPROVED** [review]

**Comment by airadier** [comment]
Move _v1 files to a /v1 folder (except the inline_scan.sh)

Add a header in the inline_scan.sh with the deprecation warning.

File: v1/Makefile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+patch:
+	$(MAKE) release INCREMENT='patch'
+
+release:
+	$(MAKE) bump INCREMENT=$(INCREMENT)
+	$(MAKE) push
+
+build:
+	docker build -t $(IMAGE_NAME):$(VERSION) -f Dockerfile . --no-cache
```
Comment (ProvoK) [inline v1/Makefile]: I think this is not gonna work since now the script file is located one directory upper

Comment (airadier) [reply v1/Makefile]: Thanks, now it should be fixed.

**Review by ProvoK — APPROVED** [review]
LGTM

**Pull request merged**